models/fates/inst/template.job

```diff
@@ -74,16 +80,17 @@ export GFORTRAN_UNBUFFERED_PRECONNECTED=yes
   
   ## ENV_BUILD update configurations
   ./xmlchange -file env_build.xml -id CIME_OUTPUT_ROOT -val @CASEDIR@
-  ./xmlchange -file env_build.xml -id EXEROOT -val @BLD@
-  ./xmlchange -file env_build.xml -id BUILD_COMPLETE -val TRUE
+  #./xmlchange -file env_build.xml -id EXEROOT -val @BLD@
+  ./xmlchange -file env_build.xml -id EXEROOT -val @CASEDIR@/bld
+  #./xmlchange -file env_build.xml -id BUILD_COMPLETE -val TRUE
```

Comment:
Unless something has changed in CIME, isn't this line required to keep CIME from completely rebuilding the model executable every time?